graph/cytoscape/cytoscape.go

```diff
@@ -346,33 +356,53 @@ func addEdgeTelemetry(ed *EdgeData, e *graph.Edge, o options.VendorOptions) {
 
 // groupByVersion adds compound nodes to group multiple versions of the same app
 func groupByVersion(nodes *[]*NodeWrapper) {
-	grouped := make(map[string][]*NodeData)
+	log.Warningf("GroupByVersion...")
```

Comment:
Minor. Is this log.Warningf() a real Warning or something more Info/Debug ?

Reply:
Thanks, you found some debug logging I left behind...